src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails.tsx

```diff
@@ -136,6 +142,17 @@ export default function FundingCycleDetails({
     ? riskWarningText.noBallot
     : riskWarningText.customBallot
 
+  const issuanceRate = isOutgoingData
+    ? deriveNextIssuanceRate({
+        weight: fundingCycle?.weight,
+        previousFC: currentFC,
+      })
+    : fundingCycle?.weight
+
+  const formattedIssuanceRate = formattedNum(
+    formatIssuanceRate(issuanceRate?.toString() ?? ''),
+  )
```

Comment:
**question** how would you feel about moving this logic to the ReconfigurationRichPreview instead? Wary of adding complexity to this component is all.

Reply:
Good idea, much cleaner

Reply:
Completed reverted this file